pyartcd/pyartcd/pipelines/images_health.py

```
@@ -83,44 +82,143 @@ async def get_report(self, version: str) -> Optional[list]:
+            'images:print',
+            '--short',
+            '{name}',
+        ]
+
+        try:
+            _, out, _ = await exectools.cmd_gather_async(cmd, stderr=None)
+            # Parse the output - one image name per line
+            valid_images = {line.strip() for line in out.strip().split('\n') if line.strip()}
+            self.runtime.logger.info('Found %d valid images for openshift-%s', len(valid_images), version)
+            return valid_images
+        except Exception as e:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)